### PR TITLE
update reenableeta

### PR DESCRIPTION
**Pull request by gebner**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by gebner** [comment]
@mattrobball There's still a couple of "-e" files here.

**Comment by mattrobball** [comment]
> @mattrobball There's still a couple of "-e" files here.

Thanks. I removed `lean-e` previously. This should remove the remainder.

**Comment by mattrobball** [comment]
Is the removable of the instance as in leanprover-community/mathlib#18986 still necessary?

**Comment by semorrison** [comment]
If it's not necessary to remove, I vote we revert the change, so we don't need to diagnose https://github.com/leanprover-community/mathlib/pull/18986.

File: Mathlib/Topology/Algebra/Module/StrongTopology.lean [hunk 1]
```diff
@@ -126,7 +126,6 @@ theorem strongTopology.topologicalAddGroup [TopologicalSpace F] [TopologicalAddG
   letI : UniformSpace (E →SL[σ] F) := strongUniformity σ F 𝔖
   haveI : UniformAddGroup (E →SL[σ] F) := strongUniformity.uniformAddGroup σ F 𝔖
   -- Porting note: this type inference is slow, so instead we provide the answer:
-  -- set_option synthInstance.maxHeartbeats 80000 in exact inferInstance
   exact UniformAddGroup.to_topologicalAddGroup
```
Comment (semorrison) [inline Mathlib/Topology/Algebra/Module/StrongTopology.lean:129]: I don't think this is the correct change.

File: Mathlib/Topology/Algebra/Module/Multilinear.lean [hunk 2]
```diff
@@ -432,7 +432,6 @@ variable [TopologicalAddGroup M₂]
 
 -- Porting note: this requires slightly more time to synthesize an instance.
 -- I couldn't see what was wrong with turning on tracing.
```
Comment (semorrison) [inline Mathlib/Topology/Algebra/Module/Multilinear.lean:434]: Should also remove the porting note?

File: Mathlib/LinearAlgebra/Matrix/Adjugate.lean [hunk 3]
```diff
@@ -502,8 +500,6 @@ theorem det_smul_adjugate_adjugate (A : Matrix n n α) :
 #align matrix.det_smul_adjugate_adjugate Matrix.det_smul_adjugate_adjugate
 
 -- Porting note: rewrites are slow.
```
Comment (semorrison) [inline Mathlib/LinearAlgebra/Matrix/Adjugate.lean:502]: Remove?

File: Mathlib/LinearAlgebra/Lagrange.lean [hunk 4]
```diff
@@ -288,8 +288,6 @@ open Finset
 
 variable {ι : Type _} [DecidableEq ι] {s t : Finset ι} {i j : ι} {v : ι → F} (r r' : ι → F)
 
-set_option maxHeartbeats 300000 in
-set_option synthInstance.maxHeartbeats 60000 in
 -- Porting note: The heartbeats were increased to find the instances needed for 'Finset.smul_sum'
 -- and 'smul_smul' on line 322.
```
Comment (semorrison) [inline Mathlib/LinearAlgebra/Lagrange.lean:292]: Remove.

File: Mathlib/Geometry/Manifold/ChartedSpace.lean [hunk 5]
```diff
@@ -307,7 +307,6 @@ structure Pregroupoid (H : Type _) [TopologicalSpace H] where
 -- failed to synthesize
 --   Subsingleton ((H → H) → Set H → Prop)
 -- (deterministic) timeout at 'typeclass', maximum number of heartbeats (20000) has been reached
```
Comment (semorrison) [inline Mathlib/Geometry/Manifold/ChartedSpace.lean:309]: Some comments to remove here.

File: Mathlib/Data/Set/Intervals/Instances.lean [hunk 6]
```diff
@@ -331,7 +331,6 @@ instance cancelMonoid {α : Type _} [StrictOrderedRing α] [IsDomain α] :
 #align set.Ioc.cancel_monoid Set.Ioc.cancelMonoid
 
 -- Porting note: This takes too long
```
Comment (semorrison) [inline Mathlib/Data/Set/Intervals/Instances.lean:333]: Remove.

File: Mathlib/Data/Matrix/Rank.lean [hunk 7]
```diff
@@ -247,7 +246,6 @@ theorem ker_mulVecLin_transpose_mul_self (A : Matrix m n R) :
 #align matrix.ker_mul_vec_lin_transpose_mul_self Matrix.ker_mulVecLin_transpose_mul_self
 
 -- Porting note: using `LinearMap.finrank_range_add_finrank_ker` is very slow
```
Comment (semorrison) [inline Mathlib/Data/Matrix/Rank.lean:248]: Remove.

**Pull request merged**